src/interactions.js

```diff
@@ -2,98 +2,118 @@
 
 var speckView = require("./view.js");
 
-module.exports = function(component, renderer, container) {
-    component.setState({
-        interactions: {
-            buttonDown: false,
-            lastX: 0.0,
-            lastY: 0.0
-        }
-    });
+module.exports = function(args) {
+    if(arguments.length > 1) {
+        throw "Error: The Speck Interactions module has changed!";
+    }
+    else if((arguments.length === 0) || (typeof args !== "object")) {
+        throw "Error: Arguments not provided to interactions";
+    }
+
+    var scrollZoom = args.scrollZoom === undefined ? true : args.scrollZoom;
```

Comment:
If `scrollZoom` is false, the zoom must be able to be set from within a Dash app. Is this the case?

Reply:
if ```var scrollZoom = false``` then that turns off scrollwheel zoom functionality entirely. However, yes the zoom can still be set by changing the `view` object in the `dash-bio` app, which is not part of the library here.